train.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import torch
+from torch import optim
+
+from dfa.model import Aligner
+from dfa.paths import Paths
+from dfa.utils import read_config, unpickle_binary
+from trainer import Trainer
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Preprocessing for DeepForcedAligner.')
+    parser.add_argument('--config', '-c', default='config.yaml', help='Points to the config file.')
+    parser.add_argument('--checkpoint', '-cp', default=None, help='Points to the a model file to restore.')
```

Comment:
How about a flag here instead? whether to train from scratch or load latest from the checkpoint dir if any?

Reply:
Could do, would you think we still need to allow pointing to checkpoints?